ui/webui/resources/BUILD.gn

```diff
@@ -305,6 +305,7 @@ leo_icons = [
   "protocol-handler-off.svg",
   "protocol-handler.svg",
   "qr-code.svg",
+  "qwant-color.svg",
```

Comment:
Did you forget to add these to the PR?

Reply:
Nah, they're already part of Nala :smile:

Reply:
https://nala.bravesoftware.com/?path=/story/components-icon--all-icons